cogs/Bartender.py

```diff
@@ -1,25 +1,72 @@
 import traceback
 import requests as req
+import random as r
 from discord.ext import commands
 from models.Drink import Drink
 
-class Bartender(commands.Cog, name='Bartender', description='Used to provide any bud with a delicious, sensual drink'):
-    FACTORY_TYPE = 'drink'
+BARTENDER_DESCRIPTION = 'Used to provide any bud with a delicious, sensual drink.'
+DRINK_DESCRIPTION = 'Get a nice, refreshing drink from the Bartender'
+DRINK_USAGE = '--> Gets a random drink\n.drink a | alcoholic --> Gets an alcoholic drink\n.drink na | nonalcoholic --> gets a nonalcoholic drink\n.drink [bourbon,brandy,cognac,gin,rum,scotch,tequila,vodka,whiskey,wine] --> Gets an alcoholic beverage of the given type'
+class Bartender(commands.Cog, name='Bartender', description=BARTENDER_DESCRIPTION):
     def __init__(self, bot):
         self.bot = bot
     
-    @commands.command(name='drink', description='Get a nice, refreshing drink from the Bartender',aliases=['drinks', 'cocktail', 'cocktails'])
-    async def drink(self, ctx):
-        DRINK_URL = 'https://www.thecocktaildb.com/api/json/v1/1/random.php'
-        try:
-            json = req.get(DRINK_URL).json()['drinks'][0]
-            drink = Drink(json)
-            await ctx.send(embed = drink.embed)
+    @commands.command(name='drink', description=DRINK_DESCRIPTION, usage=DRINK_USAGE)
+    async def drink(self, ctx, *args):
+        FILTER_TYPES = ['a','alcoholic','bourbon','brandy','cognac','gin', 'na','nonalcoholic','rum','scotch','tequila','vodka','whiskey','wine']
+        DRINK_URL = 'https://www.thecocktaildb.com/api/json/v1/1/'
+ 
+        if not args:
+            random_drink_url = DRINK_URL + 'random.php'
+            drink = await self.get_drink(ctx, random_drink_url)
+            await ctx.send(embed=drink.embed)
+            return
             
-        except Exception as ex:
+        if len(args) > 1:
+            await ctx.send(f'There can only be one argument allow for this command, bud.')
```

Comment:
```suggestion
            await ctx.send(f'There can only be one argument allowed for this command, bud.')
```
That sentence is kinda wacky tbh, but not a big deal.